thorgame/dh2540/The Power of Thor.cpp

```diff
@@ -38,47 +44,51 @@ using namespace std;
 **/
 
 int main() { 
+    // These are the four given variables.
     int light_x; // the X position of the light of power
     int light_y; // the Y position of the light of power
     int initial_tx; // Thor's starting X position
     int initial_ty; // Thor's starting Y position
     cin >> light_x >> light_y >> initial_tx >> initial_ty; cin.ignore();
 
-    // game loop
+    // Here you are given a number of turns that thor can have to reach the
+    // light. This while loop will continue until either the turns run out or
+    // thor reaches the light.
     while (1) {
         int remaining_turns; // The remaining amount of turns Thor can move. Do not remove this line.
         cin >> remaining_turns; cin.ignore();
 
-        // Write an action using cout. DON'T FORGET THE "<< endl"
-        // To debug: cerr << "Debug messages..." << endl;
-
-        // ## *My Code*
-        // If Thor's x and y coordinates are greater than the light's x and y coordinates,
-        // move Thor northwest by one.
+        // Write an action using cout. DON'T FORGET THE "<< endl" To debug: cerr
```

Comment:
This doesn't fit the rest of the comment you made -- remove or revise. I assume it was provided by the codingame site?